docs/en/02_Developer_Guides/00_Model/04_Data_Types_and_Casting.md

```diff
@@ -29,21 +29,21 @@ class Player extends DataObject
 
 ## Available Types
 
-*  [DBBoolean](api:SilverStripe\ORM\FieldType\DBBoolean): A boolean field.
```

Comment:
I think we need to list both versions, I’d suggest a format like:

```
*  `'Boolean'`: A boolean field (see [DBBoolean](api:SilverStripe\ORM\FieldType\DBBoolean))
```

Reply:
Sorry, not very familiar with the pull-request setup in github. Do I resolve the conversation here, since it's fixed in the second commit, or do you resolve the conversation as a reviewer?

Reply:
I think usually it’s the reviewer who resolves it :D